### ISSUE TITLE
EventListener Broken with 5.0.x

### ISSUE THREAD

**Issue by hjanuschka** [symfony/symfony#35259]
**Symfony version(s) affected**: 5.0.x

**Description**  

we are in the process of migrating from 4.4 to 5.0
however some of the registered event listeners, don't return a priorty - aka `null`

this happens around here:

https://sourcegraph.com/github.com/symfony/symfony@20bf17f6ad38011f90cfe9b2dc7c30c8901163e5/-/blob/src/Symfony/Component/EventDispatcher/EventDispatcher.php#L117

as the `$this->listeners[$eventName]` is nested one more times.

it results in a final exception:
![Bildschirmfoto 2020-01-08 um 13 28 26](https://user-images.githubusercontent.com/2891702/71978106-d60df980-321a-11ea-906f-0b04e3829eb6.png)

due to https://sourcegraph.com/github.com/symfony/symfony@20bf17f6ad38011f90cfe9b2dc7c30c8901163e5/-/blob/src/Symfony/Component/EventDispatcher/Debug/TraceableEventDispatcher.php#L55 - beeing type hinted with `int`

introduced with: https://github.com/symfony/symfony/commit/2bc94721b1cb844e62ccf3ffd3f7d7c3e3ee92a1

my event definition:

```yaml
services:
    krn.demo:
        class: App\EventSubscriber\Demo
        public: false
        shared: false
        tags:
          - { name: kernel.event_listener, event: krn.demo, method: onEvent }
```

**How to reproduce**  

clone: https://github.com/hjanuschka/demo
```sh
composer install
 bin/console debug:event-dispatcher krn.demo
```
<img width="545" alt="Bildschirmfoto 2020-01-08 um 13 33 02" src="https://user-images.githubusercontent.com/2891702/71978363-6f3d1000-321b-11ea-9671-4a876a0e17ec.png">

where it should show `0`

**Possible Solution**  

`return 0;` instead of `return null`  in `getListenerPriority`

PR: https://github.com/symfony/symfony/pull/35260

any help?

**Comment by OskarStark** [comment]
@derrabus can you please have a look here? Thanks 😊

**Comment by derrabus** [comment]
@OskarStark Thanks for the ping. I'll have a look.

**Comment by xabbuh** [comment]
Can you confirm that #35278 fixes this?

**Comment by hjanuschka** [comment]
i can confirm, #35278 correctly sets prio to `0` - in my trimmed-down-demo repo and in my main-application - many thx, i am closing my PR in favor of your's.

any eta, when this can be packed/released into 5.0.3?
